docs/source/tts/data/ngc_models_am.csv

```diff
@@ -8,5 +8,5 @@ en-US,RAD-TTS,TBD,TBD,TBD,ARPABET,nemo.collections.tts.models.radtts.RadTTSModel
 en-US,tts_en_tacotron2,LJSpeech,22050Hz,1,ARPABET,nemo.collections.tts.models.tacotron2.Tacotron2Model,`tts_en_tacotron2 <https://ngc.nvidia.com/catalog/models/nvidia:nemo:tts_en_tacotron2>`_,``https://api.ngc.nvidia.com/v2/models/nvidia/nemo/tts_en_tacotron2/versions/1.0.0/files/tts_en_tacotron2.nemo``
 de-DE,tts_de_fastpitch_multispeaker_5,HUI Audio Corpus German,44100Hz,5,ARPABET,nemo.collections.tts.models.fastpitch.FastPitchModel,`tts_de_fastpitch_multispeaker_5 <https://ngc.nvidia.com/catalog/models/nvidia:nemo:tts_de_fastpitch_multispeaker_5>`_,``https://api.ngc.nvidia.com/v2/models/nvidia/nemo/tts_de_fastpitch_multispeaker_5/versions/1.11.0/files/tts_de_fastpitch_multispeaker_5.nemo``
 de-DE,tts_de_fastpitch_singlespeaker,Thorsten Müller (German Neutral-TTS dataset),22050Hz,1,ARPABET,nemo.collections.tts.models.fastpitch.FastPitchModel,`tts_de_fastpitchhifigan <https://ngc.nvidia.com/catalog/models/nvidia:nemo:tts_de_fastpitchhifigan>`_,``https://api.ngc.nvidia.com/v2/models/nvidia/nemo/tts_de_fastpitchhifigan/versions/1.10.0/files/tts_de_fastpitch_align.nemo``
-es,tts_es_fastpitch_multispeaker,OpenSLR crowdsourced Latin American Spanish,44100Hz,174,grapheme,nemo.collections.tts.models.fastpitch.FastPitchModel,`tts_es_multispeaker_fastpitchhifigan <https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/tts_es_multispeaker_fastpitchhifigan>`_,``https://api.ngc.nvidia.com/v2/models/nvidia/nemo/tts_es_multispeaker_fastpitchhifigan/versions/1.14.0/files/tts_es_fastpitch_multispeaker.nemo``
+es,tts_es_fastpitch_multispeaker,OpenSLR crowdsourced Latin American Spanish,44100Hz,174,IPA,nemo.collections.tts.models.fastpitch.FastPitchModel,`tts_es_multispeaker_fastpitchhifigan <https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/tts_es_multispeaker_fastpitchhifigan>`_,``https://api.ngc.nvidia.com/v2/models/nvidia/nemo/tts_es_multispeaker_fastpitchhifigan/versions/1.15.0/files/tts_es_fastpitch_multispeaker.nemo``
```

Comment:
Why override this line? We could provide two models, one with grapheme-only and the other wit IPA.

Reply:
We could, but is there is value in maintaining 2 models? The IPA model also fully supports grapheme-only inference.

Reply:
Should we make it clearer that "IPA" means it supports mixed inputs?

Reply:
Is the quality comparable between the IPA model using graphemes only & the previous model?

If so, then no objections to replacement.

Reply:
Replacing seems fine to me, then.

> Should we make it clearer that "IPA" means it supports mixed inputs?

In this case I think that's worthwhile--we can tell users they can use any of [all graphemes/all IPA/mix].

Reply:
yeah, good question! But I vote for keeping both char and IPA ones. Char-only input is still useful for languages that lack IPA dicts, such as low-resourced African languages. 

we have char/phoneme(to be deprecated and adjusted to ipa)/ipa tokenizers for each language now, and yes we can hack IPA tokenizer to support char-only inputs, but we have to wait until detailed instructions are provided how to do char-only using ipa tokenizer.

@redoctopus what do you think?

Reply:
Honestly, I'm fine with choosing whether to replace/keep both on a case-by-case basis; if we've already done the work to determine that the new model trained on both IPA/grapheme is still comparable to the grapheme-only model for this model, then I'm not sure there's a great reason to keep the old one.

> we have char/phoneme(to be deprecated and adjusted to ipa)/ipa tokenizers for each language now, and yes we can hack IPA tokenizer to support char-only inputs, but we have to wait until detailed instructions are provided how to do char-only using ipa tokenizer.

As it stands our IPA tokenizer can still deal with grapheme input if user sets `use_chars=True`, and we currently always do this since we don't have any other way to deal with OOV. I would like to keep this functionality even in the future when we have compatibility with neural G2P that _can_ deal with OOV, so that users who still want to use a dict lookup instead can still have an option for training on samples with OOV words.

So assuming we do keep that around, we'll always technically be able to train IPA-based models that _can_ take all grapheme inputs (though without any guarantees of quality--but this one is evaluated so I think it's OK to use as a replacement).

Reply:
great discussion. approved. please help to add a sentence for this `In this case I think that's worthwhile--we can tell users they can use any of [all graphemes/all IPA/mix].`

Reply:
I do think that grapheme-only models are useful. But similar to @redoctopus I think we should handle it on a per-language or per-model basis.

Even in this case based on the data, the performance might be slightly better if I did grapheme-only + speaker pitch norm. But I would rather invest the time it takes to train and manage that model into further improving the IPA one.

The way to do the grapheme-only inference could use some improvements and documentation (eg. escaping the entire input or manually setting model.tokenizer.phoneme_probability to 0%).